cvat-data/src/ts/cvat-data.ts

```diff
@@ -253,16 +281,17 @@ export class FrameDecoder {
             releaseMutex();
         };
         try {
-            const { start, end, block } = this.requestedChunkToDecode;
-            if (start !== blockToDecode.start) {
+            const { frameNumbers, chunkNumber, block } = this.requestedChunkToDecode;
+            if (!this.arraysEqual(frameNumbers, blockToDecode.frameNumbers)) {
```

Comment:
Will not it be enough to compare `chunkNumber` and `blockToDecode.chunkNumber`?

Reply:
I think it should work.